templates/controller.tmpl

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using MVCProject.Models;
```

Comment:
I didn't tested but I think this will generate an compiler error for everybody.

Reply:
Hmm, correct, this only works in the project I use for testing. Thank you for catching this!